### PR TITLE
[FrameworkBundle] fixes outdated phpdoc on Controller::createForm() method.

### PR DESCRIPTION
**Pull request by hhamon**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php [hunk 1]
```diff
@@ -275,9 +274,9 @@ protected function createAccessDeniedException($message = 'Access Denied.', \Exc
     /**
      * Creates and returns a Form instance from the type of the form.
      *
-     * @param string|FormTypeInterface $type    The built type of the form
-     * @param mixed                    $data    The initial data for the form
-     * @param array                    $options Options for the form
+     * @param string $type    The fully qualified namespace of the form type class
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php]: What about "The fully qualified class name of the form type"?

Comment (hhamon) [inline src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php]: Sure!

Comment (hhamon) [inline src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php]: Done ;)

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Comment by nicolas-grekas** [comment]
Thank you @hhamon.

**Pull request merged**